swagger-ui/swagger-ui.js

```diff
@@ -30693,7 +30705,7 @@ Emitter.prototype.hasListeners = function(event){
  * TODO: combatible error handling?
  */
 
-module.exports = function(arr, fn, initial){  
+module.exports = function(arr, fn, initial){
```

Comment:
@howarddierking Atom purposely auto strips trailing white space. This is by design, as their argument is that code shouldn't have trailing white space in the first place. So when a file is opened in Atom, it's going to go through and remove all the trailing white spaces. Makes for a messy PR at times, but in the end I'd imagine it's a good thing?